sdk/tests/conformance/buffers/buffer-data-and-buffer-sub-data.html

```diff
@@ -76,59 +73,80 @@
     wtu.glErrorShouldBe(gl, gl.INVALID_VALUE,
         "calling bufferData when buffer size is negative should generate INVALID_VALUE");
 
-    gl.bufferData(gl.ARRAY_BUFFER, 0, gl.STATIC_DRAW);
```

Comment:
@kenrussell , it is not a bad merge.  The tests that used to be in bufferDataTestWithArrayBuffer are now part of bufferDataSizesTest. In bufferDataSizesTest, we test that, among other things, ArrayBuffers and 0 can be passed to bufferData as well as verify that the created buffer is the size we expect.